packages/editor/src/browser/editor-frontend-module.ts

```diff
@@ -30,4 +31,7 @@ export default new ContainerModule(bind => {
     bind(EditorKeybindingContext).toSelf().inSingletonScope();
     bind(KeybindingContext).toDynamicValue(context => context.container.get(EditorKeybindingContext));
     bind(KeybindingContribution).to(EditorKeybindingContribution);
+
+    bind(EditorContribution).toSelf().inSingletonScope();
+    bind(FrontendApplicationContribution).toDynamicValue(c => c.container.get(EditorContribution));
```

Comment:
No need for the dynamic value here and should be inSingletonScope()

Reply:
if someone want to access `EditorContribution`, it won't be possible

Reply:
I'm OK for the EditorContribution access part but

I'm still abit confused with:
bind(FrontendApplicationContribution).toDynamicValue(c => c.container.get(EditorContribution));

vs inSingletonScope() for that one. 

What does it means to be inSingletonScope for a contribution ?